### PR TITLE
CODEOWNERS: consolidate and update

### PR DESCRIPTION
**Pull request by jmank88**

### CODE REVIEW

**Review by RensR — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+/pkg/monitoring/ @topliceanu @pedrovsg
+
+/pkg/codec @nolag
+/pkg/loop @jmank88 @krehermann
+/pkg/services @jmank88
+/pkg/sqlutil @jmank88
+/pkg/timeutil @jmank88
+
+# Keystone
+/pkg/values/ @cedric-cordenier
```
Comment (cedric-cordenier) [inline .github/CODEOWNERS:16]: Also pkg/workflows (+@krehermann) and pkg/capabilities 🙏

**Review by vyzaldysanchez — DISMISSED** [review]

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+* @smartcontractkit/foundations
+
+/pkg/monitoring/ @topliceanu @pedrovsg
+
+/pkg/codec @nolag
+/pkg/loop @jmank88 @krehermann
+/pkg/services @jmank88
+/pkg/sqlutil @jmank88
+/pkg/timeutil @jmank88
+
```
Comment (Atrax1) [inline .github/CODEOWNERS:14]:
```suggestion
/observability-lib/ @Atrax1 
```

Comment (jmank88) [reply .github/CODEOWNERS:14]: does the trailing `/` matter? cc @patrickhuie19

File: .github/CODEOWNERS (hunk 1 above)
Comment (jmank88) [reply .github/CODEOWNERS]: Did I get it right?

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1,5 +1,17 @@
...
+/pkg/capabilities/ @cedric-cordenier @krehermann
+/pkg/codec @nolag
+/pkg/loop @jmank88 @krehermann
+/pkg/monitoring/ @topliceanu @pedrovsg
+/pkg/services @jmank88
+/pkg/sqlutil @jmank88
+/pkg/timeutil @jmank88
+/pkg/values/ @cedric-cordenier @krehermann
+/pkg/workflows/ @cedric-cordenier @krehermann
```
Comment (patrickhuie19) [inline .github/CODEOWNERS:17]: Presence of a trailing `/` will not change what is matched, but we should be consistent to eliminate confusion. I suggest we require a trailing `/`

I think it's a small enough of a change and worth to do in this PR

**Review by patrickhuie19 — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 3 above)
Comment (patrickhuie19) [reply .github/CODEOWNERS:17]: At a high level the code owner validator tool will also highlight the presence of individual ownership; we want to move to group based. 

@cedric-cordenier is there an appropriate github group for the keystone effort?

**Review by cedric-cordenier — APPROVED** [review]

File: .github/CODEOWNERS (hunk 3 above)
Comment (cedric-cordenier) [reply .github/CODEOWNERS:17]: @patrickhuie19 Yes, you could use this instead: https://github.com/orgs/smartcontractkit/teams/keystone

**Review by Atrax1 — DISMISSED** [review]

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -1,5 +1,17 @@
...
+
+/pkg/capabilities/ @cedric-cordenier @krehermann
+/pkg/codec/ @nolag
+/pkg/loop/ @jmank88 @krehermann
+/pkg/monitoring/ @topliceanu @pedrovsg
+/pkg/services/ @jmank88
+/pkg/sqlutil/ @jmank88
+/pkg/timeutil/ @jmank88
+/pkg/values/ @cedric-cordenier @krehermann
+/pkg/workflows/ @cedric-cordenier @krehermann
```
Comment (jmank88) [inline .github/CODEOWNERS:17]:
```suggestion
/pkg/capabilities/ @smartcontractkit/keystone
/pkg/codec/ @nolag
/pkg/loop/ @jmank88 @krehermann
/pkg/monitoring/ @topliceanu @pedrovsg
/pkg/services/ @jmank88
/pkg/sqlutil/ @jmank88
/pkg/timeutil/ @jmank88
/pkg/values/ @smartcontractkit/keystone
/pkg/workflows/ @smartcontractkit/keystone
```

File: .github/CODEOWNERS (hunk 3 above)
Comment (jmank88) [reply .github/CODEOWNERS]: Updated, but perhaps the team needs explicit access as a whole? 
image

**Review by Atrax1 — APPROVED** [review]

File: .github/CODEOWNERS (hunk 3 above)
Comment (jmank88) [reply .github/CODEOWNERS]: Fixed

**Review by cedric-cordenier — APPROVED** [review]

**Pull request merged**